pkgs/development/libraries/libcamera/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchgit
+, lib
+, meson
+, ninja
+, pkg-config
+, boost
+, gnutls
+, openssl
+, libevent
+, lttng-ust
+, libtiff
+, qt5
+, gst_all_1
+, graphviz
+, doxygen
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation {
+  pname = "libcamera";
+  version = "1.0.0";
+
+  src = fetchgit {
+    url = "git://linuxtv.org/libcamera.git";
+    rev = "040a6dcfbccaf1005da2c571769b1f89df1f07cf";
+    sha256 = "sha256-RCNKFCv3fNabroCUR5MtKbyDS20J7leU8KkE1U5qe1M=";
+  };
+
+  postPatch = ''
+    patchShebangs utils/
+  '';
+
+  buildInputs = [
+    # core requirements
+    python3Packages.pyyaml
+    python3Packages.ply
+    python3Packages.jinja2
+
+    # IPA and signing
+    gnutls
+    openssl
+    boost
+
+    # gstreamer integration
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+
+    # cam integration
+    libevent
+
+    # qcam integration
+    qt5.qtbase
+    qt5.qttools
+    libtiff
+
+    # lttng tracing
+    lttng-ust
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    python3
+    python3Packages.sphinx
+    graphviz
+    doxygen
+  ];
+
+  mesonFlags = [ "-Dv4l2=true" ];
+
+  dontWrapQtApps = true;
+
+  # Fixes error on a deprecated declaration
+  NIX_CFLAGS_COMPILE = [ "-Wno-error=deprecated-declarations" ];
+
+  meta = with lib; {
+    description = "An open source camera stack and framework for Linux, Android, and ChromeOS";
+    homepage = "https://libcamera.org";
+    license = licenses.bsd2;
```

Comment:
How did you determine this? I see `lgpl2Plus` in the build script: 

https://git.linuxtv.org/libcamera.git/tree/meson.build#n11

Reply:
Sorry, a mistake on my part. I've fixed this.